corehq/apps/reports/standard/cases/basic.py

```diff
@@ -111,45 +130,75 @@ def get_special_owner_ids(self, admin, unknown, demo, commtrack):
     @property
     @memoized
     def case_owners(self):
+        """
```

Comment:
I'll let @sheelio confirm, but I believe that's expected.  If you select a user or group of users (or location, which is in many ways a group of users), this report should show all cases that those users see on their phones.